performance/src/load_tests.ts

```diff
@@ -0,0 +1,53 @@
+import { JSONObject, check, sleep } from 'k6';
+import { Options, Scenario, Threshold } from 'k6/options';
+import http from "k6/http";
+import { jUnit, textSummary } from 'https://jslib.k6.io/k6-summary/0.0.2/index.js';
+
+const clusterURL = __ENV.DET_MASTER
+const masterEndpoint = '/api/v1/master';
+
+const thresholds: { [name: string]: Threshold[] } = {
+    http_req_duration: [
+        {
+            threshold: 'p(95)<1000',
+            abortOnFail: false,
+        }
+    ],
+    http_req_failed: [
+        {
+            threshold: 'rate<0.01',
+            // If more than one percent of the HTTP requests fail
+            // then we abort the test.
+            abortOnFail: true,
+        }
+    ],
+};
+
+const scenarios: { [name: string]: Scenario } = {
+    smoke: {
+        executor: 'per-vu-iterations',
+        vus: 5,
+        iterations: 250
+    },
+}
+
+export const options: Options = {
+    scenarios,
+    thresholds,
+};
+
+export default function (): void {
+    const res = http.get(`${clusterURL}${masterEndpoint}`
+    );
+    check(res, { '200 response': (r) => r.status == 200 });
+    sleep(1)
```

Comment:
do we need this one second sleep per hit?

Reply:
We do not _technically_ need the second `sleep`. However, if a `sleep` function is not invoked then the test suite will finish instantaneously. All `1250` iterations in this test will happen in less than one second. My biggest concern would be that without the sleep, the tests are not as reflective of actual API usage. Instead this test is more of a "spike" test of can the system handle hundreds of requests at once. 

I am okay removing the sleep if the plan is to create more realistic test scenarios in the future. @ashtonG

Reply:
FYI there is also https://k6.io/docs/using-k6/k6-options/reference/#rps

Reply:
Looking through k6's examples and docs, it looks like the `sleep(1)` is a common and typical setup

Reply:
yeah, i think that is why they provide a sleep func. just pointing rps out.